src/js/components/RangeInput/stories/disabled.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
Should this file be named `Disabled.js` since we typically capitalize?

Reply:
yes to match the rest I will change